migration_original/ODS1Stage/functions/Utils/generate_uuid.sql

```diff
@@ -0,0 +1,13 @@
+CREATE OR REPLACE FUNCTION generate_uuid(name VARCHAR(16777216) DEFAULT NULL)
```

Comment:
```suggestion
CREATE OR REPLACE FUNCTION ODS1_STAGE_TEAM.UTILS.generate_uuid(name VARCHAR(16777216) DEFAULT NULL)
```

Reply:
i think using another variable name is better than using name, maybe we can use code or something more descriptive

Reply:
I used name because that is the official name of the argument for version 5 UUIDs

Reply:
I did commit the change to the function name to ODS1_STAGE_TEAM.UTILS.generate_uuid